tests/test_simple.py

```diff
@@ -702,6 +702,53 @@ def test_version_management(self):
         self.assertTrue(b > c)
         self.assertTrue(a > c)
 
+    def test_pids(self):
+        try:
+            import psutil
+        except ImportError:
+            psutil = None
+
+        master_processes = (
```

Comment:
Perhaps this could be extracted as Enum (see `enums.py`) exported by `testgres`. Users might want to have strictly-typed process roles.